### PR TITLE
Simple_history for Fop, FopToKved, ExchangeData models

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/admin.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from django.contrib import admin
-from data_ocean.models.rfop_models import Rfop
+from data_ocean.models.rfop_models import Rfop, Fop, FopToKved, ExchangeData
 from data_ocean.models.ratu_models import Region, District, City, Citydistrict, Street
 from data_ocean.models.kved_models import Kved, Group, Division, Section
 
 admin.site.register(Rfop)
+admin.site.register(Fop)
+admin.site.register(FopToKved)
```
Comment (criticus) [inline data_ocean/admin.py:8]: is this needed as a separate admin page?

**Comment by criticus** [comment]
per our conversation @IegorSoboliev is going to remove FopToKved and ExchangeData from Historical records

File: data_ocean/admin.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/admin.py:8]: removed

File: data_converter/settings.py [hunk 2]
```diff
@@ -42,7 +42,8 @@
     'rest_framework',
     'rest_framework.authtoken',
     'rest_auth',
-    'django_extensions'
+    'django_extensions',
+    'simple_history',
```
Comment (criticus) [inline data_converter/settings.py:46]: does it need to be in requirements?

Comment (IegorSoboliev) [reply data_converter/settings.py:46]: Sorry, I didn`t understand you. No, that`s enough

Comment (IegorSoboliev) [reply data_converter/settings.py:46]: it is here already)

**Pull request merged**